src/app.rs

```diff
@@ -410,6 +442,7 @@ impl App {
                     .iter()
                     .map(|d| Row::new(vec![d.addr.to_string(), d.alias.to_owned()]))
                     .collect();
+                let rows_len = rows.len();
```

Comment:
Because of the padding on the help block, you need to subtract 8 to make it reach the bottom.
```
let rows_len = rows.len() - 8;
```

Reply:
Are you sure about this? That code snippet seems to belong to the rendering new devices part. I tried the same in `help.rs` but it didn't work.

Reply:
Sorry, I wanted to mentioned the same line on `help.rs`

Reply:
my bad, not here. I went little bit fast.